### ISSUE TITLE
Building for ARM

### ISSUE THREAD

**Issue by 26prajval98** [strukturag/libde265#266]
Hi, I'm trying to build the code for ARM from windows using Visual Studio 2017. 
What I did was add in the libde265 as a source to the Visual Studio Project and compiled it to get libde265.so

However, I did not include x86 folder during as it was being built for arm and sse was throwing me an error. 
Also as I'm not making the files through CMake I don't think HAVE_NEON was set and hence no modifications to the accelerators were made.

Then I tried building libheif from the generated so file. I get an unsupported codec error. I traced it down to the heif context not getting the de265 plugin.  

So my question is: 

1.  Does not including x86 files create any such errors on ARM build?
2.   Not using ARM files cause any errors in getting the decoder apart from not using it efficiently using accelerator?

**Comment by farindk** [comment]
> Does not including x86 files create any such errors on ARM build?

No. `x86` are the x86 accelerated filed, `arm` are for ARM, and it is using `fallback` otherwise.

> Not using ARM files cause any errors in getting the decoder apart from not using it efficiently using accelerator?

No. It works fine with just the `fallback` routines and no acceleration code.
